configure.sh

```diff
@@ -74,6 +74,22 @@ function check_build_type() {
     esac
 }
 
+function check_assets() {
```

Comment:
not needed, please remove - it's not like we add all options to configure & catalog directory.
Also - there is no need to - reconfiguration should be lightweight for this example.

Reply:
removed all changes on configure.sh layer